CONTRIBUTING.md

```diff
@@ -0,0 +1,39 @@
+# Contributing to mmcif_pdbx
+
+We are excited to have you contribute to APBS!
+If you have general questions about the code, please see the [documentation](https://mmcif-pdbx.readthedocs.io/) and/or start a thread in the [discussion forum](https://github.com/Electrostatics/mmcif_pdbx/discussions). 
```

Comment:
**discussions** not yet activated? It's a dead link.

Reply:
Sorry; should be active now.

Reply:
@sobolevnrm Did you want to create the standard `Welcome` post like the one from [APBS](https://github.com/Electrostatics/apbs/discussions/133)?